Reintroduce the Quick effect selectors in the EQ preferences

daschuer opened the pull request:
We need that to merge the effects_refactoring branch without a regression until the Skin effect GUI is in place. https://bugs.launchpad.net/mixxx/+bug/1948537

This does not revert to the 2.4 state where single effects where selected, It selects the effect chain presets and is fully aligned with the upcoming chain preset selectors in the skins.

This was already closed by @Be-ing without giving appropriate reasons. 
https://github.com/mixxxdj/mixxx/pull/4485

I still think this is a good step forward and takes some pressure form all of us. 
Please read also my reasons form https://bugs.launchpad.net/mixxx/+bug/1948537

What do other think?

Be-ing reviewed (changes requested):
Adding technical debt instead of actually solving problems is not okay. It is really disappointing you chose to waste time on this instead of helping.

daschuer committed:
Reintroduce the Quick effect selectors in the EQ preferences

We need that to merge the effects_refactoring branch without a regression until the Skin effect GUI is in place. https://bugs.launchpad.net/mixxx/+bug/1948537

Be-ing commented:
I accept that I have been outvoted, but I can't help point out the unfair treatment here. When @daschuer is the only one opposed to merging something, he has been allowed to hold up everything for weeks or months, for example #4239 or #2920, sometimes without even an explanation (#2618).

Swiftb0y commented:
I can understand that you feel treated unfairly. We're trying to change this by overruling vetos more often by voting. This is one of the first PRs where we're trying this. I understand this seems biased right now, but you'll see that this is not the case when we do this for more and more PRs.
